File: LLaMA3.1-405B/Python/2-3.py
def file_read_write():
    source_file = input("Enter the source file name: ")
    destination_file = input("Enter the destination file name: ")

    try:
        with open(source_file, 'r') as file:
            content = file.read()

        with open(destination_file, 'w') as file:
            file.write(content)

        print("Content copied successfully!")

    except FileNotFoundError:
        print("Source file not found!")

file_read_write()

File: LLaMA3.1-405B/C++/2-3.c
#include <iostream>
#include <fstream>
using namespace std;

int main() {
    string sourceFile, destinationFile;

    cout << "Enter the source file name: ";
    cin >> sourceFile;

    cout << "Enter the destination file name: ";
    cin >> destinationFile;

    ifstream source(sourceFile);
    ofstream destination(destinationFile);

    if (source.is_open() && destination.is_open()) {
        string line;
        while (getline(source, line)) {
            destination << line << endl;
        }
        source.close();
        destination.close();
        cout << "Content copied successfully!" << endl;
    } else {
        cout << "Error opening file!" << endl;
    }

    return 0;
}

